usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -164,44 +176,184 @@ def handle(self, *args, **options):
+        self.logger = logging.getLogger("script")
 
         # Resolve Parameters
-        delta_table = options["delta_table"]
-        recreate = options["recreate"]
-
-        table_spec = TABLE_SPEC[delta_table]
+        self.options = options
+        source_delta_table = self.options["delta_table"]
+        self.table_spec = TABLE_SPEC[source_delta_table]
 
         # Delta side
-        destination_database = options["alt_delta_db"] or table_spec["destination_database"]
-        delta_table_name = options["alt_delta_name"] or delta_table
-        delta_table = f"{destination_database}.{delta_table_name}" if destination_database else delta_table_name
-
-        # Postgres side - source
-        postgres_table = None
-        postgres_model = table_spec["model"]
-        postgres_schema = table_spec["source_database"] or table_spec["swap_schema"]
-        postgres_table_name = table_spec["source_table"] or table_spec["swap_table"]
-        postgres_cols = table_spec["source_schema"]
-        column_names = table_spec.get("column_names")
-        tsvectors = table_spec.get("tsvectors") or {}
-        if postgres_table_name:
-            postgres_table = f"{postgres_schema}.{postgres_table_name}" if postgres_schema else postgres_table_name
-
-        # Postgres side - temp
-        temp_schema = "temp"
-        temp_table_suffix = "temp"
-        temp_table_suffix_appendage = f"_{temp_table_suffix}" if {temp_table_suffix} else ""
-        if postgres_table:
-            temp_table_name = f"{postgres_table_name}{temp_table_suffix_appendage}"
-        else:
-            temp_table_name = f"{delta_table_name}{temp_table_suffix_appendage}"
-        temp_table = f"{temp_schema}.{temp_table_name}"
+        source_delta_table_name = self.options["alt_delta_name"] or source_delta_table
+        source_delta_database = self.options["alt_delta_db"] or self.table_spec["destination_database"]
+        self.qualified_source_delta_table = f"{source_delta_database}.{source_delta_table_name}"
+        delta_table_load_version_key = self.table_spec["delta_table_load_version_key"]
+
+        # Postgres side
+        postgres_schema = self.table_spec["swap_schema"]
+        postgres_schema_def = self.table_spec["source_schema"]
+        self.postgres_table_name = self.table_spec["swap_table"]
+        self.qualified_postgres_table = f"{postgres_schema}.{self.postgres_table_name}"
+        postgres_temp_schema = "temp"
+
+        # Determine the latest version of the Delta Table at the time this command is running. This will later
+        # be stored as the latest version written to the staging table, regardless of whether we update incrementally
+        # or repopulate the entire table
+        last_update_version = (
+            spark.sql(f"DESCRIBE HISTORY {self.qualified_source_delta_table}").select("version").first()[0]
+        )
+
+        updated_incrementally = False
+
+        if self.options["incremental"]:
```

Comment:
Is `self.options["incremental"]` a Boolean? If not I think this if statement could be made more clear.